examples/chat/tests/pages/ChatPage.ts

```diff
@@ -0,0 +1,33 @@
+import { Page, Locator, expect } from "@playwright/test";
+
+export class ChatPage {
+  readonly page: Page;
+  readonly messageInput: Locator;
+  readonly logoutButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.messageInput = page.getByPlaceholder("Type a message and press Enter");
```

Comment:
I think ideally we should label the field rather than using the placeholder (at a guess aria-label would be most appropriate here)

Reply:
Thanks for the suggestion!

Done with a sr-only label.